allocate.go

```diff
@@ -75,7 +78,8 @@ func (c *Cluster) allocate(hash *cid.Cid, rplMin, rplMax int, blacklist []peer.I
 		rplMin,
 		rplMax,
 		currentMetrics,
-		candidatesMetrics)
+		candidatesMetrics,
+		priorityMetrics)
```

Comment:
I am not sure if there is a current convention in ipfs codebases, but I usually follow the same pattern as a multiline struct initialization for multiline function calls, see below:
```go
s := &stuff{
    value1,
    value2,
}

s.DoStuff(
   param1,
   param2,
)
```
This way you don't have a two line addition from `git`'s perspective when you have a single line addition. Also, all parameters have the same indentation level. I often find my eyes skip over the first parameter if it is on the same line as the function call, and then all other params listed on consecutive lines.

Please feel free to disagree :)

Reply:
I agree with this.  @hsanjuan is this already the convention or if not is it something you agree on making a convention in cluster code?

Reply:
I'm happy with trying to maintain this convention, but I'm also ok as long as code is `gofmt`'ed.